students/eunchan/westagram/users/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+
+# Create your models here.
+
+
+class Member(models.Model):
+    name = models.CharField(max_length=50)
+    email = models.CharField(max_length=100, unique=True)
+    password = models.CharField(max_length=200)
+    phone_num = models.CharField(max_length=20)
+    person_info = models.CharField(max_length=200)
```

Comment:
변수명을 줄여쓰지 않도록 주의해주세요!
```suggestion
    information = models.CharField(max_length=200)
```